foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/oxm/record/json/JsonStructureReader.java

```diff
@@ -449,7 +453,25 @@ private void parsePair(String name, JsonValue jsonValue) throws SAXException {
                 if(attributePrefix == null && nv !=null ){
                   return;
                 }
-            }
+			} else if (isUnmarshalRecordWithinAdapter()) {
+				@SuppressWarnings("rawtypes")
+				final UnmarshalRecord contentHandler_ = getUnmarshalRecordFromAdapter();
+				if (jsonTypeCompatibility) {
```

Comment:
looks like 'tab's are used here instead of '4 spaces'; can you fix formatting, please?